### ISSUE TITLE
SpriteSheet: allow specifying image padding

### ISSUE THREAD

**Issue by LearnCocos2D** [spritebuilder/SpriteBuilder#1056]
In certain use cases it can be important to have the images in the sprite sheet be a certain distance away from each other to prevent drawing parts of other images.

Case in point, notice the "stray bit of texture" just above the player:

![screen shot 2014-10-23 at 14 39 50](https://cloud.githubusercontent.com/assets/214599/4752941/bed517ee-5ab1-11e4-9b8e-8e1d731c374d.png)

The soft-body renderer takes the texture based on the bodies drawn around the body. Since this is a circular body it can easily happen that parts of a neighboring texture are drawn as well. I can't make the radius smaller though, otherwise parts of the texture may be cut off. Being able to increase the image padding would fix that problem.

**Comment by cocojoe** [comment]
If you have a look in the Tupac lib I believe the padding is exposed, worth a manual test to check it actually does work as expected.

**Comment by LearnCocos2D** [comment]
I still can't wrap my head around why anyone would call that lib "tupac". Anyhow, I'll see if I can find this in the code.

**Comment by LearnCocos2D** [comment]
Yes, increasing `padding` in the tupac instance helps to reduce/prevent this particular problem.  It also caused some sheets to grow too large now, so it definitely has an effect.
